superset/connectors/sqla/views.py

```diff
@@ -270,107 +269,15 @@ class SqlMetricInlineView(  # pylint: disable=too-many-ancestors
     edit_form_extra_fields = add_form_extra_fields
 
 
-class RowLevelSecurityListWidget(
-    SupersetListWidget
-):  # pylint: disable=too-few-public-methods
-    template = "superset/models/rls/list.html"
```

Comment:
@michael-s-molina We can also remove this html file.

Reply:
Good point. Removed.